### PR TITLE
ai-gen: Update metadata

### PR DESCRIPTION
**Pull request by josephschmitt**
## Description
Doing some wordsmithing so the extension is easier to find in the store if searching for "GPT-3" or "DALL•E"

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/ai-gen/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ai-gen: OpenAI Image and Text Generator
+# OpenAI Image and Text Generator
 
 The `ai-gen` extension uses the OpenAI [Image Generation](https://beta.openai.com/docs/guides/images) API — colloquially known as [DALL·E 2](https://openai.com/dall-e-2/) — and the [Text Completion](https://beta.openai.com/docs/guides/completion) API to generate results from a text-based prompt.
```
Comment (peduarte) [inline extensions/ai-gen/README.md:3]: Should the description be updated?

```suggestion
The OpenAI Generator extension uses the OpenAI [Image Generation](https://beta.openai.com/docs/guides/images) API — colloquially known as [DALL·E 2](https://openai.com/dall-e-2/) — and the [Text Completion](https://beta.openai.com/docs/guides/completion) API to generate results from a text-based prompt.
```

**Comment by peduarte** [comment]
@josephschmitt Love the new wording 👌

Are you still happy with the `ai-gen`? This applies to the directory (`/extensions/ai-gen`) and `package.name` property. Wondering if it's worth changing it to `openai-generator`. If so, I could handle the renaming internally, so it'll be a seamless change for users. 

Let me know what you think.

**Comment by josephschmitt** [comment]
@peduarte I don't really care, to be honest. It seems like that name doesn't actually show up anywhere users see. I'm fine leaving it as-is unless you feel there's a compelling reason to rename

**Comment by peduarte** [comment]
@josephschmitt I don't mind. It just helps with discoverability within the `extensions` folder. Typically the folder name will be pretty much the same as the Extension name, so if someone wants to find your extension, they can easily find it.

**Comment by josephschmitt** [comment]
Integrated these changes into #3471

**Pull request closed without merging**